### ISSUE TITLE
Notification gets sent multiple times

### ISSUE THREAD

**Issue by Lgmrszd** [modrunner/discord-bot#120]
### What happened?

I have setup this bot to post notification for my mod updates. But sometimes when I release a new version, notification gets sent multiple times.
I've checked (`/list`) that I didn't accidentally add same project twice and that's not the case.

**Comment by smcmo** [comment]
Would you be able to attach a screenshot here?

**Comment by Lgmrszd** [comment]
Sure but I don't think there's anything useful on here, it's just same messages
![image](https://github.com/modrunner/discord-bot/assets/16291541/d0094579-2dc7-4b34-a53a-b0eb45c22b0b)
![image](https://github.com/modrunner/discord-bot/assets/16291541/34c64319-24fc-4f75-8162-3c5439fb1da0)

**Comment by Bowswa** [comment]
Hello! I am here because this just started happening in three discord i am in & run. All versions after the issue started occurring are posted twice, and when the issue first appeared it found the latest version of the followed project and re-posted it. This has happened on 6/26/24 & 6/27/24 so far.

**Comment by Bowswa** [comment]
> Would you be able to provide the name(s) of the servers, and the name(s) of the project(s) where this has been happening?

Yes. I can provide invite links so you can see whatever details you need:
https://discord.gg/ntRb7Nw7Xt
https://discord.gg/N6XcaUt7

And project Links:
https://modrinth.com/datapack/mo-variants
https://modrinth.com/modpack/imaginary
https://modrinth.com/datapack/blood-moon
https://modrinth.com/datapack/injured-effects

And a currently Unlisted/Private project called "Masters Season 2 Modpack"

**Comment by seriousfreezing** [comment]
I'm suffering from the same bug. Apparently even the Simply Optimized server is suffering from the same thing

**Comment by smcmo** [comment]
> I'm suffering from the same bug. Apparently even the Simply Optimized server is suffering from the same thing

Hello, yes. I spoke with Bowswa in DMs and we figured out that this happened due to some instability with Modrinth's API. Any servers tracking Modrinth projects would have been affected by this.

I'm working on adding additional checks into the bot's notification process to help prevent duplicate notifications from showing up at random again.

**Comment by Lgmrszd** [comment]
Is there any update on this? It is marked as closed but the issue still persists
